### ISSUE TITLE
Test model for face recognition

### ISSUE THREAD

**Issue by arad2022** [anjith2006/bob.paper.tbiom2023_edgeface#2]
How I can test pre-trained model for face recognition?
Could you please provide any code example?

I test the below code but I don't know how should I compare two embeddings?

import torch
from torchvision import transforms
# from face_alignment import align
from backbones import get_model
from PIL import Image
from scipy.spatial.distance import cosine

arch="edgeface_s_gamma_05" # or edgeface_xs_gamma_06
model=get_model(arch)

transform = transforms.Compose([
            transforms.ToTensor(),
            transforms.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
            ])

checkpoint_path=r'D:\face recognition\code\edgeface\bob.paper.tbiom2023_edgeface-master\checkpoints\edgeface_s_gamma_05.pth'
model.load_state_dict(torch.load(checkpoint_path))
model.eval()
path = 'checkpoints/synthface.jpeg'

# aligned = align.get_aligned_face(path)
img = Image.open(path).convert('RGB')
transformed_input = transform(img).unsqueeze(0)
embedding = model(transformed_input)

@anjith2006
